include/boost/geometry/policies/robustness/segment_ratio.hpp

```diff
@@ -53,8 +53,10 @@ struct less<Type, false>
     {
         BOOST_GEOMETRY_ASSERT(lhs.denominator() != 0);
         BOOST_GEOMETRY_ASSERT(rhs.denominator() != 0);
-        return lhs.numerator() * rhs.denominator()
-             < rhs.numerator() * lhs.denominator();
+        Type const l = lhs.numerator() / lhs.denominator();
```

Comment:
Please avoid using `l` as a variable name

Reply:
Is `a` and `b` clear enough?

Reply:
Sure, fine for me.